lib/apimatic-core/logger/configurations/api_logging_configuration.rb

```diff
@@ -38,25 +38,25 @@ class BaseMessageLoggingConfiguration
     # @param headers_to_unmask [Array<String>] Array of headers which values are non-sensitive to display in logging.
     #   Default is an empty array.
     def initialize(
-      log_body: false,
-      log_headers: false,
-      headers_to_exclude: [],
-      headers_to_include: [],
-      headers_to_unmask: []
+      log_body: nil,
```

Comment:
Question: Since all parameter default values are nil so can we make them all required?